app/views/shared/_content_card.html.erb

```diff
@@ -5,6 +5,18 @@
                else
                  nil
                end %>
+<% status = if item.status.present?
```

Comment:
I think this chaining is not working. I'm not seeing any content_dates even if the status is complete. Status should be mandatory (I've added a check to ensure that in this branch). So we don't need to check for it's presence, just it's value.

Also if there's no content_type the status shows in the left, instead of the right.

One other thing that I just noticed, but is missing. Activities can also have status and content type, they just won't have a content_date.

Lots of rules, I know! Let me know if they are not clear!